app/views/landing/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="header-color-bar"> </div>
+
+<div class="logo-section">
+  <%= image_tag 'landing/phone.png'%>
+  <div class="vertical-separator"> </div>
+  <%= image_tag 'landing/tcg_logo.png'%>
+</div>
+
+<hr />
```

Comment:
The triangles needed to style this part of the landing, should be done with CSS pseudo elements (https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements), since this is not an actual element of the view it's just there for decoration purposes.